README.md

```diff
@@ -1,9 +1,15 @@
 ![Open Service Broker API](https://www.openservicebrokerapi.org/wp-content/uploads/2016/12/osbapi_logo_concept3_wtm.png)
 
-[The Open Service Broker API specification](spec.md)
+## Latest Release: v2.11
+[Read the specification](https://github.com/openservicebrokerapi/servicebroker/tree/v2.11)
 
 [Release Notes for past versions](release-notes.md)
 
+## Working Changes
```

Comment:
Correct - I see this as anything that has been merged into master ready for the next release. However, I wanted to call out that we still reserve the right to change things before the official release.